pyCRISPRcleanR/plots.py

```diff
@@ -194,39 +200,40 @@ def roc_curve(df, data_type='', fdrth=0.05, saveto='./roc_curve'):
         :param saveto:
         :return:
         """
-        df, roc_auc, sens, _ = PlotData._roc_curve_r(df.tf.values, df.avgFC.values)
 
-        recall = df.sensitivity.values
-        tnr = df.specificity.values
+        df, roc_auc, sens, _ = PlotData._roc_curve_r(df.tf.values, df.avgFC.values)
+        if not np.isnan(_):
```

Comment:
Is underscore being used as a normal variable here? I didn't know this could be done. The convention is to use underscore for return values that we don't care about. In this case it would seem that we don't care about the fourth return value, but then we are doing a test for it.